db/schema.rb

```diff
@@ -114,8 +114,11 @@
   create_table "schools", :force => true do |t|
     t.string  "name"
     t.integer "campus_id"
-    t.string  "code",      :limit => 8
-    t.boolean "active",                 :default => false
+    t.string  "code",        :limit => 8
```

Comment:
Is this the zcode?  I think the zcodes need to be a foreign key to the zcode mapping table.  Let's talk on slack, if need.

Reply:
@daryanypl in this PR, the only thing that changed on this line is the indentation of `:limit`.  `code` remains unchanged.  To answer your question, here's an example value of a school's code in the database: "zm610".